src/models/darp8/repairs.md

```diff
@@ -0,0 +1,211 @@
+# Darter Pro (Parts & Repairs)
+
+Many components in your Darter Pro can be upgraded or replaced as necessary. Follow these step-by-step guides for instructions:
+
+- [Removing the bottom cover](#removing-the-bottom-cover)
+- [Replacing the RAM](#replacing-the-ram)
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Replacing the WiFi/Bluetooth module](#replacing-the-wireless-card)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the keyboard](#replacing-the-keyboard)
+- [Replacing the battery](#replacing-the-battery)
+
+## Removing the bottom cover:
+
+Removing the cover is required to access the internal components. Prior to removing the cover, ensure the AC power is unplugged and all peripherals (including SD cards and USB drives) are unplugged or removed from the system.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 7 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to remove the bottom cover:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 16 bottom panel screws:
+    - 12 perimeter screws (thin), highlighted green below.
+    - 2 keyboard screws (thicker), highlighted yellow.
+    - 2 inner hinge screws (thickest), highlighted red.
```

Comment:
The hinge and keyboard screws are the same size on darp8

Reply:
I wasn't 100% sure looking at them in person, must have been my imagination making me think they were still different. I'll update.

Reply:
I didn't skimp out on updating the colors in the image, but upon further consideration, the keyboard screws are obviously good to be able to locate easily (even though it's called out again in the keyboard section), and it could be useful to know that the remaining two are for the hinges. With that in mind, I kept them colored differently, and just updated the wording to describe both as "thick" compared to the perimeter screws being "thin."

Let me know if you disagree, I can still update the colors if we want the four thicker ones to be colored the same despite their differing functions.

Reply:
Yeah that makes sense to me